src/app/tests/suites/certification/Test_TC_CNET_4_12.yaml

```diff
@@ -180,34 +180,13 @@ tests:
       PICS: CNET.S.C06.Rsp
```

Comment:
@raul-marquez-csa TC-CNET-4.12: Script has to be updated from Step 8 to 20 as per Verification steps. We have a sample app issue in Step 8 to 20 so marked as red color. But the Verification steps are correct.

Reply:
Updated